build-configs/BuildConfigType.js

```diff
@@ -3,11 +3,21 @@
 import type { ThemeType } from './ThemeType'
 import type { TranslationsType } from 'translations'
 
+// Prevent enabled intro slide in combination with a preselected city.
+// If you change this make sure you are not navigating to the landing screen upon closing the intro slides.
+export type SelectedCityType = {|
+  introSlides: false,
+  selectedCity: string | null // If set, the city is automatically preselected. Changing it is not possible for users.
```

Comment:
Should we name this `preselectedCity` maybe? Or `fixedCity`?

Reply:
This could case confusion with the name in our State which is also `selectedCity`

Reply:
I thought about `preselectedCity` as well, the reason I did not went for it is that it sounds like it is selected on first start but can be changed afterwards. I'm fine with `fixedCity` though.